src/particles/ParticleContainer.js

```diff
@@ -149,7 +149,11 @@ export default class ParticleContainer extends core.Container
             this._properties[1] = 'position' in properties ? !!properties.position : this._properties[1];
             this._properties[2] = 'rotation' in properties ? !!properties.rotation : this._properties[2];
             this._properties[3] = 'uvs' in properties ? !!properties.uvs : this._properties[3];
-            this._properties[4] = 'alpha' in properties ? !!properties.alpha : this._properties[4];
+            this._properties[4] = ('alpha' in properties) ? !!properties.alpha : this._properties[4];
+            if ('tint' in properties)
+            {
+                this._properties[4] = !!properties.tint;
+            }
```

Comment:
I don't think this logic is quite right, if I set `alpha` to true, but `tint` to false then alpha doesn't get uploaded. Probably need to change this to something like:

```js
this._properties[4] = 'alpha' in properties || 'tint' in properties ? !!properties.alpha || !!properties.tint : this._properties[4];
```

That way either of them being set will use the value, and either of them being set to true will cause the upload.

Reply:
Agree.